create-table.md

```diff
@@ -1,22 +1,111 @@
 ---
 title: CREATE TABLE
 toc: false
+toc_not_nested: true
```

Comment:
I know the contributing.md file asked for `toc_not_nested` at some point, but it's no longer needed.